debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/AbstractReplicationMessageColumn.java

```diff
@@ -140,6 +158,10 @@ public OptionalInt getScale() {
             return scale != null ? OptionalInt.of(scale) : OptionalInt.empty();
         }
 
+        public String[] getModifiers() {
```

Comment:
Why do we expose the raw modifiers? My hope ws the more dedicated methods (`getLength()` etc.) would suffice, as they are more explicit and descriptive.

Reply:
Reasoning was that any custom types used can have any arbitrary modifiers. eg. Postgis types use `GEOMETRY(Polygon, 12345)` — "polygon" is the geometry type and "12345" is the coordinate system reference. Neither of those map to length/etc terms.